A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -23,19 +23,19 @@ class CDUIdentPage {
         mcdu.page.Current = mcdu.page.IdentPage;
         mcdu.activeSystem = 'FMGC';
         mcdu.setTemplate([
-            ["A320-200"],
+            ["A320-251N"],
             ["\xa0ENG"],
-            ["LEAP 1A-26[color]green"],
+            ["LEAP-1A26[color]green"],
             ["\xa0ACTIVE NAV DATA BASE"],
-            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "AIRAC[color]green"],
+            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "FBW2014001[color]green"],
             ["\xa0SECOND NAV DATA BASE"],
-            ["{small}{04MAY-04JUL{end}[color]cyan"],
-            ["", "STORED"],
-            ["", "{green}10{end}{small}RTES\xa0{end}{green}10{end}{small}RWYS{end}"],
-            ["CHG CODE", "[b-text]{big}{green}20{end}{end}WPTS\xa0{big}{green}20{end}{end}NAVS"],
-            ["{small}[  ]{end}[color]cyan", "DELETE ALL}[color]cyan"],
```

Comment:
Can this be put back?

Reply:
is there a way to add it back but still show this not being there. because in the real plane it is just blank